apps/webapp/src/app/(Magic-Link)/magic-link/page.tsx

```
@@ -0,0 +1,614 @@
+                You&apos;ve successfully connected your account!
+              </span>
+            </div>
+            <button
+              onClick={() => {
+                CloseSuccessDialog(false);
+                router.replace("/connections");
+                // window.location.href = config.WEBAPP_URL;
+              }}
+              className="mt-4 px-6 py-2 bg-black text-white rounded-lg font-semibold hover:bg-gray-100 hover:text-black transition-colors"
+            >
+              Go to Dashboard
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)